common/src/main/java/com/microsoft/identity/common/crypto/AndroidWrappedKeyLoader.java

```diff
@@ -206,7 +206,8 @@ protected SecretKey generateRandomKey() throws ClientException {
             final byte[] wrappedSecretKey = FileUtil.readFromFile(getKeyFile(), KEY_FILE_SIZE);
             if (wrappedSecretKey == null) {
                 Logger.warn(methodTag, "Key file is empty");
-                deleteSecretKeyFromStorage();
+                FileUtil.deleteFile(getKeyFile());
```

Comment:
Can you add some comments in the code (for future readers) why this case must be treated differently from the other one.